storybookjs/storybook
# Fix for issue 5932, adding default options to action(s)

Issue: https://github.com/storybooks/storybook/issues/5932

Apologies for any missteps, first time doing a pull for a open source project :).

## What I did
I noticed that both `action` and `actions` defaulted their options to `{}`, hence the reported issue. To remedy this, I imported `config` from `configureActions.ts` and set it as the new default for ActionOptions.

## How to test
To test, you can play around with the “with text” and “with some emoji” stories in the cra-kitchen-sink example story (https://github.com/storybooks/storybook/tree/next/examples/cra-kitchen-sink). 

Sample test flow:
1. Navigate to “Button” -> “with text” in the cra-kitchen-sink storybook
2. Click the “Hello Button” a couple times to generate some action logs.
3. Switch to the “with some emoji” story. Note that the action logs aren’t wiped since that action doesn’t have `clearOnStoryChange: true`
4. Click the emoji button in the “with some emoji” story. Note that a new action is logged
5. Change to a different story. Note that the action log is wiped clean since the action associated with that button doesn’t have any options, thus using the default config from `configureActions`.

Additional testing scenarios can be done by adding `configureActions` with various settings in the `config.js` of the cra-kitchen-sink storybook.

- Is this testable with Jest or Chromatic screenshots?
- Does this need a new example in the kitchen sink apps?
- Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.
